src/server/kernel/woutputlayout.h

```diff
@@ -18,7 +19,10 @@ class WAYLIB_SERVER_EXPORT WOutputLayout : public QW_NAMESPACE::QWOutputLayout,
 public:
     enum class Layer {
         Background = -999,
-        Cursor = 999
+        Cursor = INT_MAX
```

Comment:
`INT_MAX - 1000`, reserved some values after Cursor layer.

Reply:
z is qreal,  `INT_MAX+1.0` is still available
Of course use `INT_MAX - 1000` is also acceptable